Directory.Build.targets

```diff
@@ -50,6 +50,16 @@
     <ExcludeByFile></ExcludeByFile>
   </PropertyGroup>
 
+  <!-- Global Analyzer Config -->
+  <ItemGroup>
+    <!-- Always include Common.globalconfig -->
+    <EditorConfigFiles Include="$(RepositoryEngineeringDir)globalconfigs\Common.globalconfig" />
+    <!-- Include Shipping.globalconfig for shipping projects -->
+    <EditorConfigFiles Condition="'$(IsShipping)' == 'true'" Include="$(RepositoryEngineeringDir)globalconfigs\Shipping.globalconfig" />
+    <!-- Include NonShipping.globalconfig for non-shipping projects -->
+    <EditorConfigFiles Condition="'$(IsShipping)' != 'true'" Include="$(RepositoryEngineeringDir)globalconfigs\NonShipping.globalconfig" />
```

Comment:
Are we accidentally setting `IsShipping` for `Test.Utilities`? NonShipping.globalconfig disables `CA1034`, but it seems `Test.Utilities` still has `CA1034` violations.

Reply:
I think for all test projects, we get the correct value for free from Arcade:

https://github.com/dotnet/arcade/blob/3003926e4126f827bca50d5b3ee179afc86d8a7b/src/Microsoft.DotNet.Arcade.Sdk/tools/Tests.props#L15-L18

But since `Test.Utilities` doesn't end end "Tests", we don't get that. I'll set `IsUnitTestProject` manually for that project.

Reply:
That didn't work for some reason, so I set `<IsShipping>false</IsShipping>` directly.